util/common.go

```diff
@@ -488,6 +488,19 @@ func Exists(path string) bool {
 	return true
 }
 
+// IsDir returns true if the path is a directory.
+func IsDir(path string) bool {
+	info, err := os.Stat(path)
+	if err != nil {
+		return false
+	}
+
+	if info.IsDir() {
+		return true
+	}
+	return false
```

Comment:
Very tiny nit, this can be a little simplified, but I'm not going to block on it:
```suggestion
	return info.IsDir()
```